pom.xml

```diff
@@ -5,7 +5,7 @@
   <parent>
     <artifactId>schemas</artifactId>
     <groupId>org.geonetwork-opensource</groupId>
-    <version>3.7</version>
+    <version>3.10-SNAPSHOT</version>
```

Comment:
Because of this change we need to wait for the artifact build to be back ported to 3.10.x :(